recipe/meta.yaml

```diff
@@ -19,18 +19,18 @@ build:
         - vc14  # [win and py35]
 
 build:
-    number: 1
+    number: 2
 
 requirements:
     build:
         - python  # [win]
         - cmake  # [win]
         - zlib
-        - jpeg
+        - jpeg 8d
```

Comment:
Apologies. Happy to move to the latest version so long as it is pinned *correctly* 😄